lib/matplotlib/tests/test_afm.py

```diff
@@ -2,10 +2,45 @@
 
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
+from six import BytesIO
 
 import matplotlib.afm as afm
 
 
+AFM_TEST_DATA = b"""StartFontMetrics 2.0
```

Comment:
Can we use one of the afm files in https://github.com/matplotlib/matplotlib/tree/master/lib/matplotlib/mpl-data/fonts ?  I am (with out thinking about it too much) concerned about copyright on including this in the source.

Is there something about this afm file that triggers the bug that does not trigger it otherwise?

Reply:
These files are a bit lengthy in their CharMetrics block, which would make it a bit cumbersome to test their char metrics. I've changed the `AFM_TEST_DATA` to a shorter one with invented data. That should prevent any copyright issues.